syncd/PortMap.cpp

```diff
@@ -0,0 +1,51 @@
+#include "swss/logger.h"
+#include "PortMap.h"
+
+/**
+ * @brief Port map global map.
+ *
+ * WARNING: This object must have global declaration and this exact name since
+ * external RPC server is linking against this object when in use.
+ */
+std::map<std::set<int>, std::string> gPortMap;
+
+void PortMap::insert(
+        _In_ const std::set<int>& laneSet,
+        _In_ const std::string& name)
+{
+    SWSS_LOG_ENTER();
+
+    m_portMap.insert(std::make_pair(laneSet, name));
```

Comment:
`m_portMap.emplace(laneSet, name);`
for my taste looks better

Reply:
that was copy paste from original code :)